jack.py

```diff
@@ -674,6 +675,38 @@ def transport_stop(self):
         """Stop JACK transport."""
         _lib.jack_transport_stop(self._ptr)
 
+    def set_timebase_callback(self,
+                              conditional, timebase_callback, userdata=None):
+        """Register as timebase master for the JACK subsystem.
+
+        The timebase master registers a callback that updates extended position
+        information such as beats or timecode whenever necessary. Without this
+        extended information, there is no need for this function.
+
+        There is never more than one master at a time. When a new client takes
+        over, the former timebase_callback is no longer called. Taking over the
+        timebase may be done conditionally, so it fails if there was a master
+        already.
+
+        The method may be called whether the client has been activated or not.
+
+        Parameters
+        ----------
+        conditional : int
+            non-zero for a conditional request.
+        timebase_callback : callable
```

Comment:
Please add documentation about the function signature, like in the other callback setters.

You should also copy the relevant text from `types.h`.